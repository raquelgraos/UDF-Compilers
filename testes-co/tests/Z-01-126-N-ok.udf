int f(int a) { return a + 1; }
public int udf() {
  int i = 9;
  auto t = [[2+1, i], [f(i), 2]];
  writeln t;
  return 0;
}

