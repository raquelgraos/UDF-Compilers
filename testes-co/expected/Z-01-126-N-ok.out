Tensor<2,2>[[3, 9], [1E1, 2]]
